apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -203,21 +206,23 @@ function AssessmentQuestionsTable({
                 : ''}
               <tr>
                 <td>
-                  <a href="${urlPrefix}/question/${question.question_id}/">
-                    ${question.alternative_group_size === 1
-                      ? `${question.alternative_group_number}.`
-                      : html`
-                          <span class="ml-3">
-                            ${question.alternative_group_number}.${question.number_in_alternative_group}.
-                          </span>
-                        `}
-                    ${question.title}
-                    ${IssueBadge({
-                      urlPrefix,
-                      count: question.open_issue_count ?? 0,
-                      issueQid: question.qid,
-                    })}
-                  </a>
+                  ${hasCoursePermissionPreview
+                    ? unsafeHtml(`<a href="${urlPrefix}/question/${question.question_id}/">`)
```

Comment:
```suggestion
                    ? html`<a href="${urlPrefix}/question/${question.question_id}/">`
```

Reply:
I tried that (see my earlier commit "Fixed prettier issue with non-closed link"), but then prettier complains that the tag is not closed.

Reply:
Ah, that makes sense. As a rule, `unsafeHtml` should be reserved for exceptional cases. If I can recommend an alternative: this might be a good fit for the `run` function from https://github.com/PrairieLearn/PrairieLearn/pull/10595.

```ts
html`
  ${run(() => {
    const number = question.alternative_group_size === 1
      ? `${question.alternative_group_number}.`
      : html`
          <span class="ml-3">
            ${question.alternative_group_number}.${question.number_in_alternative_group}.
          </span>
        `;
    const title = html`${number} ${question.title}`;

    if (hasCoursePermissionPreview) {
      return html`<a href="${urlPrefix}/question/${question.question_id}/">${title}</a>`;
    }

    return title;
  })}
`;
```

Plus or minus some Prettier formatting, of course.

Reply:
Nice, this certainly is handy here (I understand that unsafeHtml is not ideal, but without run the alternative would've been even uglier...)